src/libraries/System.Private.Xml.Linq/src/System/Xml/Linq/XDocumentType.cs

```diff
@@ -12,14 +12,14 @@ namespace System.Xml.Linq
     public class XDocumentType : XNode
     {
         private string _name;
-        private string _publicId;
-        private string _systemId;
+        private string? _publicId;
+        private string? _systemId;
         private string _internalSubset;
 
         /// <summary>
         /// Initializes an empty instance of the <see cref="XDocumentType"/> class.
         /// </summary>
-        public XDocumentType(string name, string publicId, string systemId, string internalSubset)
+        public XDocumentType(string name, string? publicId, string? systemId, string internalSubset)
```

Comment:
most likely `internalSubset` should be nullable (looking at other classes and no guarding against null)

Reply:
Did you notice comment in line https://github.com/dotnet/runtime/pull/40744/files#diff-2b58a7484d2af6925596bd988eae2a42R60-R63?

>As per documentation, this should return string.Empty.

Reply:
didn't notice 😄